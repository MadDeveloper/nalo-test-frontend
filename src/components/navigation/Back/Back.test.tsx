import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { Back } from "."

describe("Components > Back", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Back to="/" />
        </BrowserRouter>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
