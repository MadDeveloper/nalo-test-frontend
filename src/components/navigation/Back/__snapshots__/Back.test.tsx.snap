// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Back should match snapshot 1`] = `
<a
  className="root"
  href="/"
  onClick={[Function]}
>
  <svg
    aria-hidden="true"
    className="icon"
    fill="none"
    stroke="currentColor"
    strokeWidth={1.5}
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
  <span
    className="label"
  >
    Retour
  </span>
</a>
`;
